polyfills/Array/from/polyfill.js

```diff
@@ -1,3 +1,44 @@
+
+function parseIterable(arraylike) {
+	var
+	done = false,
+	iterableResponse,
+	tempArray;
```

Comment:
Ah I was following the style already existing in the polyfill I will update that accordingly.